content/blog/production-ready-object-detection-model-training-workflow-with-hpe-machine-learning-development-environment.md

```diff
@@ -0,0 +1,1107 @@
+* Create a Persistent Volume Claim for local model deployment
+* Preparing custom model for Kserve inference
+* Deploying model using a KServe InferenceService
+* Complete a sample request and plot predictions
+
+Note: This notebook was tested on a Linux-based machine with Nvidia T4 GPUs. We also assume Docker is installed in your Linux system/environment
+
+Let's get started!
+
+## Pre-reqs: Setting up Python and Jupyter Lab environment
+
+Run the below commands to set up a python virtual environment, and install all the python packages needed for this tutorial
+
+```
+sudo apt-get update && sudo apt-get  install python3.8-venv
+python3 -m venv kserve_env
+source kserve_env/bin/activate
+pip install kserve jupyterlab torch-model-archiver
+pip install torch==1.11.0 torchvision==0.12.0 matplotlib
+jupyter lab --ip=0.0.0.0 \
+  --port=8008 \
+  --NotebookApp.token='' \
+  --NotebookApp.password=''
+```
+
+## Install Kserve Natively using Kind and Knative
+
+### Install Kind
+
+Open a Terminal and run the following bash commands to install a kubernetes cluster using Kind:
+
+* `curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.18.0/kind-linux-amd64`
+* `chmod +x ./kind`
+* `sudo mv ./kind /usr/local/bin/kind`
+
+After running these commands, create a cluster by running the command: `kind create cluster`
+
+### Install Kubectl
+
+Run the following bash commmands in a terminal to to install the kubectl runtime:
+
+* `curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"`
+* `curl -LO "https://dl.k8s.io/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl.sha256"`
+* `sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl`
+
+### Install Kserve
+
+Run this bash script to install KServe onto our default Kubernetes Cluster, note this will install the following artifacts:
+
+* ISTIO_VERSION=1.15.2, KNATIVE_VERSION=knative-v1.9.0, KSERVE_VERSION=v0.9.0-rc0, CERT_MANAGER_VERSION=v1.3.0
+* `bash e2e_blogposts/ngc_blog/kserve_utils/bash_scripts/kserve_install.sh`
+
+### Patch Domain for local connection to KServe cluster/environment
+
+Run this command to patch your cluster when you want to connect to your cluster on the same machine:
+
+`kubectl patch cm config-domain --patch '{"data":{"example.com":""}}' -n knative-serving`
+
+### Run Port Forwarding to access KServe cluster
+
+* `INGRESS_GATEWAY_SERVICE=$(kubectl get svc --namespace istio-system --selector="app=istio-ingressgateway" --output jsonpath='{.items[0].metadata.name}')`
+* `kubectl port-forward --namespace istio-system svc/${INGRESS_GATEWAY_SERVICE} 8080:80`
+
+Make sure to open a new terminal to continue the configuration. 
+
+### Create a Persistent Volume Claim for local model deployment
+
+We will be creating a Persistent Volume Claim to host and access our Pytorch based Object Detection model locally. A persistent volume claim requires three k8s artifacts:
```

Comment:
Change to:
"You will be creating a persistent volume claim to host and access the PyTorch-based object detection model locally. "
Second sentence is fine.